config/kafka-rest.properties

```diff
@@ -17,6 +17,7 @@
 #id=kafka-rest-test-server
 #schema.registry.url=http://localhost:8081
 #zookeeper.connect=localhost:2181
+bootstrap.servers=PLAINTEXT://localhost:9092
```

Comment:
This used to be a default value but we want users to explicitly specify. Specifying this in the config  will make the quick start easy.